railties/test/application/test_runner_test.rb

```diff
@@ -597,14 +597,14 @@ def test_parallel_is_enabled_when_PARALLEL_WORKERS_is_set
 
         class ParallelTest < ActiveSupport::TestCase
           def test_verify_test_order
-            puts "Test order: \#{self.class.test_order}"
+            puts "Test parallelization disabled: \#{ActiveSupport.test_parallelization_disabled}"
           end
         end
       RUBY
 
       output = run_test_command(file_name)
 
-      assert_match "Test order: parallel", output
+      assert_match "Test order: random", output
```

Comment:
@leequarella as per https://github.com/ricardotk002/rails/commit/fc5581ab this bit should be:

```suggestion
      assert_match "Test parallelization disabled: false", output
```